boost 1.86 causes build failures

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

This issue is present in the latest pre-release

### Describe the Bug

Windows builds are failing after updating boost to 1.86 in msys2.

### Expected Behavior

_No response_

### Additional Context

https://github.com/msys2/MINGW-packages/commit/d29d34fe44838545392c787bca5bd7c25f303910

### Host Operating System

Windows

### Operating System Version

any

### Architecture

64 bit

### Sunshine commit or version

beta

### Package

n/a

### GPU Type

n/a

### GPU Model

n/a

### GPU Driver/Mesa Version

n/a

### Capture Method

None

### Config

```Shell
n/a
```


### Apps

_No response_

### Relevant log output

```shell
FAILED: CMakeFiles/sunshine.dir/src/main.cpp.obj 
D:\a\_temp\msys64\ucrt64\bin\c++.exe -DBOOST_ATOMIC_NO_LIB -DBOOST_CHRONO_NO_LIB -DBOOST_FILESYSTEM_NO_LIB -DBOOST_LOCALE_NO_LIB -DBOOST_LOG_NO_LIB -DBOOST_PROGRAM_OPTIONS_NO_LIB -DBOOST_SYSTEM_NO_LIB -DBOOST_THREAD_NO_LIB -DCURL_STATICLIB -DMINIUPNP_STATICLIB -DNDEBUG -DSUNSHINE_ASSETS_DIR=\"assets\" -DSUNSHINE_PLATFORM=\"windows\" -DSUNSHINE_PUBLISHER_ISSUE_URL=\"https://app.lizardbyte.dev/support\" -DSUNSHINE_PUBLISHER_NAME=\"LizardByte\" -DSUNSHINE_PUBLISHER_WEBSITE=\"https://app.lizardbyte.dev\" -DSUNSHINE_TRAY=1 -DUSE_BOOST_REGEX -ID:/a/Sunshine/Sunshine -ID:/a/Sunshine/Sunshine/build -isystem D:/a/Sunshine/Sunshine/third-party/build-deps/ffmpeg/Windows-AMD64/include -isystem D:/a/Sunshine/Sunshine/third-party/nanors/deps/obl -isystem D:/a/Sunshine/Sunshine/third-party/nanors -isystem D:/a/Sunshine/Sunshine/third-party/moonlight-common-c/enet/include -isystem D:/a/Sunshine/Sunshine/third-party -isystem D:/a/Sunshine/Sunshine/third-party/nv-codec-headers/include -isystem D:/a/Sunshine/Sunshine/third-party/nvapi-open-source-sdk -isystem D:/a/Sunshine/Sunshine/third-party/ViGEmClient/include -O2 -g -DNDEBUG -std=gnu++20 -Wall -Wno-sign-compare -Wno-maybe-uninitialized -Werror -Wno-misleading-indentation -MD -MT CMakeFiles/sunshine.dir/src/main.cpp.obj -MF CMakeFiles\sunshine.dir\src\main.cpp.obj.d -o CMakeFiles/sunshine.dir/src/main.cpp.obj -c D:/a/Sunshine/Sunshine/src/main.cpp
In file included from D:/a/Sunshine/Sunshine/src/process.h:17,
                 from D:/a/Sunshine/Sunshine/src/main.cpp:19:
D:/a/Sunshine/Sunshine/src/platform/common.h:55:13: error: reference to 'basic_environment' is ambiguous
   55 |     typedef basic_environment<char> environment;
      |             ^~~~~~~~~~~~~~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/traits/wchar_t.hpp:14,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/basic_cmd.hpp:14,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/args.hpp:33,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:8,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process.hpp:27,
                 from D:/a/Sunshine/Sunshine/src/process.h:14:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/traits/env.hpp:15:7: note: candidates are: 'template<class Char> class boost::process::v1::basic_environment'
   15 | class basic_environment;
      |       ^~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:54:11: note:                 'template<class Char> class boost::process::basic_environment'
   54 |     class basic_environment;
      |           ^~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:600:3: error: reference to 'child' is ambiguous
  600 |   boost::process::child
      |   ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/child.hpp:21,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/async_system.hpp:22,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:10:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:110:5: error: reference to 'child' is ambiguous
  110 |     boost::process::child _process;
      |     ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:111:5: error: reference to 'group' is ambiguous
  111 |     boost::process::group _process_group;
      |     ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:11:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/group.hpp:61:7: note: candidates are: 'class boost::process::v1::group'
   61 | class group
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:52:11: note:                 'class boost::process::group'
   52 |     class group;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:3: error: variable or field 'terminate_process_group' declared void
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |   ^~~~~~~~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:43: error: reference to 'child' is ambiguous
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                           ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:54: error: expected primary-expression before ',' token
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                      ^
D:/a/Sunshine/Sunshine/src/process.h:146:72: error: reference to 'group' is ambiguous
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                        ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/group.hpp:61:7: note: candidates are: 'class boost::process::v1::group'
   61 | class group
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:52:11: note:                 'class boost::process::group'
   52 |     class group;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: error: 'group' was not declared in this scope
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                               ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: note: suggested alternatives:
In file included from D:/a/_temp/msys64/ucrt64/include/boost/format.hpp:48,
                 from D:/a/Sunshine/Sunshine/src/stat_trackers.h:11,
                 from D:/a/Sunshine/Sunshine/src/logging.h:24,
                 from D:/a/Sunshine/Sunshine/src/main.cpp:16:
D:/a/_temp/msys64/ucrt64/include/boost/format/group.hpp:679:3: note:   'boost::io::group'
  679 |   group(T1 a1,T2 a2,T3 a3,T4 a4,T5 a5,T6 a6,T7 a7,T8 a8,T9 a9, Var& var)
      |   ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: note:   'boost::process::v1::group, boost::process::group'
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                               ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:107: error: expected primary-expression before 'exit_timeout'
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                                                           ^~~~~~~~~~~~
[22/136] Building CXX object CMakeFiles/sunshine.dir/src/logging.cpp.obj
[23/136] Building CXX object CMakeFiles/sunshine.dir/src/httpcommon.cpp.obj
FAILED: CMakeFiles/sunshine.dir/src/httpcommon.cpp.obj 
D:\a\_temp\msys64\ucrt64\bin\c++.exe -DBOOST_ATOMIC_NO_LIB -DBOOST_CHRONO_NO_LIB -DBOOST_FILESYSTEM_NO_LIB -DBOOST_LOCALE_NO_LIB -DBOOST_LOG_NO_LIB -DBOOST_PROGRAM_OPTIONS_NO_LIB -DBOOST_SYSTEM_NO_LIB -DBOOST_THREAD_NO_LIB -DCURL_STATICLIB -DMINIUPNP_STATICLIB -DNDEBUG -DSUNSHINE_ASSETS_DIR=\"assets\" -DSUNSHINE_PLATFORM=\"windows\" -DSUNSHINE_PUBLISHER_ISSUE_URL=\"https://app.lizardbyte.dev/support\" -DSUNSHINE_PUBLISHER_NAME=\"LizardByte\" -DSUNSHINE_PUBLISHER_WEBSITE=\"https://app.lizardbyte.dev\" -DSUNSHINE_TRAY=1 -DUSE_BOOST_REGEX -ID:/a/Sunshine/Sunshine -ID:/a/Sunshine/Sunshine/build -isystem D:/a/Sunshine/Sunshine/third-party/build-deps/ffmpeg/Windows-AMD64/include -isystem D:/a/Sunshine/Sunshine/third-party/nanors/deps/obl -isystem D:/a/Sunshine/Sunshine/third-party/nanors -isystem D:/a/Sunshine/Sunshine/third-party/moonlight-common-c/enet/include -isystem D:/a/Sunshine/Sunshine/third-party -isystem D:/a/Sunshine/Sunshine/third-party/nv-codec-headers/include -isystem D:/a/Sunshine/Sunshine/third-party/nvapi-open-source-sdk -isystem D:/a/Sunshine/Sunshine/third-party/ViGEmClient/include -O2 -g -DNDEBUG -std=gnu++20 -Wall -Wno-sign-compare -Wno-maybe-uninitialized -Werror -Wno-misleading-indentation -MD -MT CMakeFiles/sunshine.dir/src/httpcommon.cpp.obj -MF CMakeFiles\sunshine.dir\src\httpcommon.cpp.obj.d -o CMakeFiles/sunshine.dir/src/httpcommon.cpp.obj -c D:/a/Sunshine/Sunshine/src/httpcommon.cpp
In file included from D:/a/Sunshine/Sunshine/src/process.h:17,
                 from D:/a/Sunshine/Sunshine/src/httpcommon.cpp:7:
D:/a/Sunshine/Sunshine/src/platform/common.h:55:13: error: reference to 'basic_environment' is ambiguous
   55 |     typedef basic_environment<char> environment;
      |             ^~~~~~~~~~~~~~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/traits/wchar_t.hpp:14,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/basic_cmd.hpp:14,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/args.hpp:33,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:8,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process.hpp:27,
                 from D:/a/Sunshine/Sunshine/src/process.h:14:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/traits/env.hpp:15:7: note: candidates are: 'template<class Char> class boost::process::v1::basic_environment'
   15 | class basic_environment;
      |       ^~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:54:11: note:                 'template<class Char> class boost::process::basic_environment'
   54 |     class basic_environment;
      |           ^~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:600:3: error: reference to 'child' is ambiguous
  600 |   boost::process::child
      |   ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/child.hpp:21,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/async_system.hpp:22,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:10:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:110:5: error: reference to 'child' is ambiguous
  110 |     boost::process::child _process;
      |     ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:111:5: error: reference to 'group' is ambiguous
  111 |     boost::process::group _process_group;
      |     ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:11:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/group.hpp:61:7: note: candidates are: 'class boost::process::v1::group'
   61 | class group
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:52:11: note:                 'class boost::process::group'
   52 |     class group;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:3: error: variable or field 'terminate_process_group' declared void
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |   ^~~~~~~~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:43: error: reference to 'child' is ambiguous
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                           ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:54: error: expected primary-expression before ',' token
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                      ^
D:/a/Sunshine/Sunshine/src/process.h:146:72: error: reference to 'group' is ambiguous
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                        ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/group.hpp:61:7: note: candidates are: 'class boost::process::v1::group'
   61 | class group
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:52:11: note:                 'class boost::process::group'
   52 |     class group;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: error: 'group' was not declared in this scope
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                               ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: note: suggested alternatives:
D:/a/Sunshine/Sunshine/src/process.h:146:79: note:   'boost::process::v1::group, boost::process::group'
In file included from D:/a/_temp/msys64/ucrt64/include/boost/format.hpp:48,
                 from D:/a/Sunshine/Sunshine/src/stat_trackers.h:11,
                 from D:/a/Sunshine/Sunshine/src/logging.h:24,
                 from D:/a/Sunshine/Sunshine/src/platform/common.h:20:
D:/a/_temp/msys64/ucrt64/include/boost/format/group.hpp:679:3: note:   'boost::io::group'
  679 |   group(T1 a1,T2 a2,T3 a3,T4 a4,T5 a5,T6 a6,T7 a7,T8 a8,T9 a9, Var& var)
      |   ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:107: error: expected primary-expression before 'exit_timeout'
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                                                           ^~~~~~~~~~~~
[24/136] Installing node modules

added 1 package, changed 1 package, and audited 3 packages in 59s

found 0 vulnerabilities
[25/136] Building CXX object CMakeFiles/sunshine.dir/src/nvhttp.cpp.obj
FAILED: CMakeFiles/sunshine.dir/src/nvhttp.cpp.obj 
D:\a\_temp\msys64\ucrt64\bin\c++.exe -DBOOST_ATOMIC_NO_LIB -DBOOST_CHRONO_NO_LIB -DBOOST_FILESYSTEM_NO_LIB -DBOOST_LOCALE_NO_LIB -DBOOST_LOG_NO_LIB -DBOOST_PROGRAM_OPTIONS_NO_LIB -DBOOST_SYSTEM_NO_LIB -DBOOST_THREAD_NO_LIB -DCURL_STATICLIB -DMINIUPNP_STATICLIB -DNDEBUG -DSUNSHINE_ASSETS_DIR=\"assets\" -DSUNSHINE_PLATFORM=\"windows\" -DSUNSHINE_PUBLISHER_ISSUE_URL=\"https://app.lizardbyte.dev/support\" -DSUNSHINE_PUBLISHER_NAME=\"LizardByte\" -DSUNSHINE_PUBLISHER_WEBSITE=\"https://app.lizardbyte.dev\" -DSUNSHINE_TRAY=1 -DUSE_BOOST_REGEX -ID:/a/Sunshine/Sunshine -ID:/a/Sunshine/Sunshine/build -isystem D:/a/Sunshine/Sunshine/third-party/build-deps/ffmpeg/Windows-AMD64/include -isystem D:/a/Sunshine/Sunshine/third-party/nanors/deps/obl -isystem D:/a/Sunshine/Sunshine/third-party/nanors -isystem D:/a/Sunshine/Sunshine/third-party/moonlight-common-c/enet/include -isystem D:/a/Sunshine/Sunshine/third-party -isystem D:/a/Sunshine/Sunshine/third-party/nv-codec-headers/include -isystem D:/a/Sunshine/Sunshine/third-party/nvapi-open-source-sdk -isystem D:/a/Sunshine/Sunshine/third-party/ViGEmClient/include -O2 -g -DNDEBUG -std=gnu++20 -Wall -Wno-sign-compare -Wno-maybe-uninitialized -Werror -Wno-misleading-indentation -MD -MT CMakeFiles/sunshine.dir/src/nvhttp.cpp.obj -MF CMakeFiles\sunshine.dir\src\nvhttp.cpp.obj.d -o CMakeFiles/sunshine.dir/src/nvhttp.cpp.obj -c D:/a/Sunshine/Sunshine/src/nvhttp.cpp
In file included from D:/a/Sunshine/Sunshine/src/nvhttp.cpp:32:
D:/a/Sunshine/Sunshine/src/process.h:71:34: error: expected ')' before '&&' token
   71 |       boost::process::environment &&env,
      |                                  ^~~
      |                                  )
D:/a/Sunshine/Sunshine/src/process.h:70:11: note: to match this '('
   70 |     proc_t(
      |           ^
D:/a/Sunshine/Sunshine/src/process.h:102:5: error: reference to 'environment' is ambiguous
  102 |     boost::process::environment _env;
      |     ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/env.hpp:10,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:14,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process.hpp:27,
                 from D:/a/Sunshine/Sunshine/src/process.h:14:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/environment.hpp:650:37: note: candidates are: 'typedef class boost::process::v1::basic_environment<char> boost::process::v1::environment'
  650 | typedef basic_environment<char>     environment;
      |                                     ^~~~~~~~~~~
In file included from D:/a/Sunshine/Sunshine/src/nvhttp.cpp:31:
D:/a/Sunshine/Sunshine/src/platform/common.h:55:37: note:                 'typedef class boost::process::basic_environment<char> boost::process::environment'
   55 |     typedef basic_environment<char> environment;
      |                                     ^~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/process.h:110:5: error: reference to 'child' is ambiguous
  110 |     boost::process::child _process;
      |     ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/child.hpp:21,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/async_system.hpp:22,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:10:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:111:5: error: reference to 'group' is ambiguous
  111 |     boost::process::group _process_group;
      |     ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:11:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/group.hpp:61:7: note: candidates are: 'class boost::process::v1::group'
   61 | class group
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:52:11: note:                 'class boost::process::group'
   52 |     class group;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:3: error: variable or field 'terminate_process_group' declared void
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |   ^~~~~~~~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:43: error: reference to 'child' is ambiguous
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                           ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:54: error: expected primary-expression before ',' token
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                      ^
D:/a/Sunshine/Sunshine/src/process.h:146:72: error: reference to 'group' is ambiguous
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                        ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/group.hpp:61:7: note: candidates are: 'class boost::process::v1::group'
   61 | class group
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:52:11: note:                 'class boost::process::group'
   52 |     class group;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: error: 'group' was not declared in this scope
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                               ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: note: suggested alternatives:
In file included from D:/a/_temp/msys64/ucrt64/include/boost/format.hpp:48,
                 from D:/a/Sunshine/Sunshine/src/stat_trackers.h:11,
                 from D:/a/Sunshine/Sunshine/src/logging.h:24,
                 from D:/a/Sunshine/Sunshine/src/nvhttp.cpp:28:
D:/a/_temp/msys64/ucrt64/include/boost/format/group.hpp:679:3: note:   'boost::io::group'
  679 |   group(T1 a1,T2 a2,T3 a3,T4 a4,T5 a5,T6 a6,T7 a7,T8 a8,T9 a9, Var& var)
      |   ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: note:   'boost::process::v1::group, boost::process::group'
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                               ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:107: error: expected primary-expression before 'exit_timeout'
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                                                           ^~~~~~~~~~~~
[26/136] Building CXX object CMakeFiles/sunshine.dir/src/confighttp.cpp.obj
FAILED: CMakeFiles/sunshine.dir/src/confighttp.cpp.obj 
D:\a\_temp\msys64\ucrt64\bin\c++.exe -DBOOST_ATOMIC_NO_LIB -DBOOST_CHRONO_NO_LIB -DBOOST_FILESYSTEM_NO_LIB -DBOOST_LOCALE_NO_LIB -DBOOST_LOG_NO_LIB -DBOOST_PROGRAM_OPTIONS_NO_LIB -DBOOST_SYSTEM_NO_LIB -DBOOST_THREAD_NO_LIB -DCURL_STATICLIB -DMINIUPNP_STATICLIB -DNDEBUG -DSUNSHINE_ASSETS_DIR=\"assets\" -DSUNSHINE_PLATFORM=\"windows\" -DSUNSHINE_PUBLISHER_ISSUE_URL=\"https://app.lizardbyte.dev/support\" -DSUNSHINE_PUBLISHER_NAME=\"LizardByte\" -DSUNSHINE_PUBLISHER_WEBSITE=\"https://app.lizardbyte.dev\" -DSUNSHINE_TRAY=1 -DUSE_BOOST_REGEX -ID:/a/Sunshine/Sunshine -ID:/a/Sunshine/Sunshine/build -isystem D:/a/Sunshine/Sunshine/third-party/build-deps/ffmpeg/Windows-AMD64/include -isystem D:/a/Sunshine/Sunshine/third-party/nanors/deps/obl -isystem D:/a/Sunshine/Sunshine/third-party/nanors -isystem D:/a/Sunshine/Sunshine/third-party/moonlight-common-c/enet/include -isystem D:/a/Sunshine/Sunshine/third-party -isystem D:/a/Sunshine/Sunshine/third-party/nv-codec-headers/include -isystem D:/a/Sunshine/Sunshine/third-party/nvapi-open-source-sdk -isystem D:/a/Sunshine/Sunshine/third-party/ViGEmClient/include -O2 -g -DNDEBUG -std=gnu++20 -Wall -Wno-sign-compare -Wno-maybe-uninitialized -Werror -Wno-misleading-indentation -MD -MT CMakeFiles/sunshine.dir/src/confighttp.cpp.obj -MF CMakeFiles\sunshine.dir\src\confighttp.cpp.obj.d -o CMakeFiles/sunshine.dir/src/confighttp.cpp.obj -c D:/a/Sunshine/Sunshine/src/confighttp.cpp
In file included from D:/a/Sunshine/Sunshine/src/process.h:17,
                 from D:/a/Sunshine/Sunshine/src/confighttp.cpp:9:
D:/a/Sunshine/Sunshine/src/platform/common.h:55:13: error: reference to 'basic_environment' is ambiguous
   55 |     typedef basic_environment<char> environment;
      |             ^~~~~~~~~~~~~~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/traits/wchar_t.hpp:14,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/basic_cmd.hpp:14,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/args.hpp:33,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:8,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process.hpp:27,
                 from D:/a/Sunshine/Sunshine/src/process.h:14:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/traits/env.hpp:15:7: note: candidates are: 'template<class Char> class boost::process::v1::basic_environment'
   15 | class basic_environment;
      |       ^~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:54:11: note:                 'template<class Char> class boost::process::basic_environment'
   54 |     class basic_environment;
      |           ^~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:600:3: error: reference to 'child' is ambiguous
  600 |   boost::process::child
      |   ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/child.hpp:21,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1/async_system.hpp:22,
                 from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:10:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:110:5: error: reference to 'child' is ambiguous
  110 |     boost::process::child _process;
      |     ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:111:5: error: reference to 'group' is ambiguous
  111 |     boost::process::group _process_group;
      |     ^~~~~
In file included from D:/a/_temp/msys64/ucrt64/include/boost/process/v1.hpp:11:
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/group.hpp:61:7: note: candidates are: 'class boost::process::v1::group'
   61 | class group
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:52:11: note:                 'class boost::process::group'
   52 |     class group;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:3: error: variable or field 'terminate_process_group' declared void
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |   ^~~~~~~~~~~~~~~~~~~~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:43: error: reference to 'child' is ambiguous
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                           ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/detail/child_decl.hpp:45:7: note: candidates are: 'class boost::process::v1::child'
   45 | class child
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:51:11: note:                 'class boost::process::child'
   51 |     class child;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:54: error: expected primary-expression before ',' token
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                      ^
D:/a/Sunshine/Sunshine/src/process.h:146:72: error: reference to 'group' is ambiguous
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                        ^~~~~
D:/a/_temp/msys64/ucrt64/include/boost/process/v1/group.hpp:61:7: note: candidates are: 'class boost::process::v1::group'
   61 | class group
      |       ^~~~~
D:/a/Sunshine/Sunshine/src/platform/common.h:52:11: note:                 'class boost::process::group'
   52 |     class group;
      |           ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: error: 'group' was not declared in this scope
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                               ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:79: note: suggested alternatives:
D:/a/Sunshine/Sunshine/src/process.h:146:79: note:   'boost::process::v1::group, boost::process::group'
In file included from D:/a/_temp/msys64/ucrt64/include/boost/format.hpp:48,
                 from D:/a/Sunshine/Sunshine/src/stat_trackers.h:11,
                 from D:/a/Sunshine/Sunshine/src/logging.h:24,
                 from D:/a/Sunshine/Sunshine/src/platform/common.h:20:
D:/a/_temp/msys64/ucrt64/include/boost/format/group.hpp:679:3: note:   'boost::io::group'
  679 |   group(T1 a1,T2 a2,T3 a3,T4 a4,T5 a5,T6 a6,T7 a7,T8 a8,T9 a9, Var& var)
      |   ^~~~~
D:/a/Sunshine/Sunshine/src/process.h:146:107: error: expected primary-expression before 'exit_timeout'
  146 |   terminate_process_group(boost::process::child &proc, boost::process::group &group, std::chrono::seconds exit_timeout);
      |                                                                                                           ^~~~~~~~~~~~
ninja: build stopped: subcommand failed.
Error: Process completed with exit code 1.
```

--- a/.codeql-prebuild-cpp-Windows.sh
+++ b/.codeql-prebuild-cpp-Windows.sh
@@ -19,7 +19,6 @@ pacman -Syu --noconfirm --ignore=mingw-w64-ucrt-x86_64-curl \
   gcc \
   git \
   make \
-  mingw-w64-ucrt-x86_64-boost \
   mingw-w64-ucrt-x86_64-cmake \
   mingw-w64-ucrt-x86_64-cppwinrt \
   mingw-w64-ucrt-x86_64-graphviz \

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -1033,7 +1033,6 @@ jobs:
           pacman -Syu --noconfirm \
             --ignore=mingw-w64-ucrt-x86_64-curl \
             git \
-            mingw-w64-ucrt-x86_64-boost \
             mingw-w64-ucrt-x86_64-cmake \
             mingw-w64-ucrt-x86_64-cppwinrt \
             mingw-w64-ucrt-x86_64-graphviz \
@@ -1144,6 +1143,7 @@ jobs:
             --exclude-unreachable-branches \
             --exclude '.*tests/.*' \
             --exclude '.*third-party/.*' \
+            --exclude '.*_deps/.*' \
             --xml-pretty \
             -o coverage.xml
 

